locale/zh_CN.po

```diff
@@ -6790,32 +6791,32 @@ msgstr "右键"
 #: ../src/propgrid/advprops.cpp:1673
 msgctxt "system cursor name"
 msgid "Sizing NE-SW"
-msgstr "缩放 (东北至西南)"
+msgstr "尺寸 NE-SW"
 
 #: ../src/propgrid/advprops.cpp:1674
 msgctxt "system cursor name"
 msgid "Sizing N-S"
-msgstr "缩放 (南至北)"
+msgstr "尺寸 N-S"
 
 #: ../src/propgrid/advprops.cpp:1675
 msgctxt "system cursor name"
 msgid "Sizing NW-SE"
-msgstr "缩放 (东南至西北)"
+msgstr "尺寸 NW-SE"
 
 #: ../src/propgrid/advprops.cpp:1676
 msgctxt "system cursor name"
 msgid "Sizing W-E"
-msgstr "缩放 (东至西)"
+msgstr "尺寸 W-S"
```

Comment:
These are actually taken from the `zh-TW.po`. Any reasons to revert these?

Reply:
I had no idea... zh_TW.po looks like this
![Screen Shot 2022-07-06 at 2 16 18 AM](https://user-images.githubusercontent.com/82932766/177390614-8ffcd5a1-ab09-490d-a887-47a5604e60ea.png)

Reply:
> Thanks for the great work @obmij. Overall LGTM. There are some minor issues that I commented inline. Most others are nitpicking. You can either take it or leave it as you see fit (my comments are not 100% correct either). Thanks again.



> Thanks for the great work @obmij. Overall LGTM. There are some minor issues that I commented inline. Most others are nitpicking. You can either take it or leave it as you see fit (my comments are not 100% correct either). Thanks again.

@hjiawei Pleasure collaboration! How about giving me some time to dig out these symbols and pieces of stuff. I assume it's no guilt for deferring a few days? ;-)